index.js

```diff
@@ -25,47 +49,71 @@ function makeCopier(copyConfig, assetManager, { compact } = {}) {
 		filter: copyConfig.filter
 	});
 	let { fingerprint } = copyConfig;
-	let plugins = determinePlugins(compact, copyConfig);
+	let compactors = determineCompactors(compact, copyConfig);
 
 	return filepaths => {
 		return Promise.all([
 			(filepaths ? fileFinder.match(filepaths) : fileFinder.all()),
 			determineTargetDir(source, target)
 		]).then(([fileNames, targetDir]) => {
 			return processFiles(fileNames, {
-				assetManager, source, target, targetDir, plugins, fingerprint
+				assetManager, source, target, targetDir, compactors, fingerprint
 			});
 		});
 	};
 }
 
-function determinePlugins(compact, copyConfig) {
+/**
+ * Determine which compactors should be used
+ *
+ * @param {boolean | undefined} compact
+ * @param {Config} copyConfig
+ * @returns {CompactorMap}
+ */
+function determineCompactors(compact, copyConfig) {
 	if(!compact) {
 		return {};
 	}
 
 	return copyConfig.compact || {};
 }
 
-// If `source` is a directory, `target` is used as target directory -
-// otherwise, `target`'s parent directory is used
+/**
+ * If `source` is a directory, `target` is used as target directory -
+ * otherwise, `target`'s parent directory is used
+ *
+ * @param {string} source
+ * @param {string} target
+ * @returns {Promise<string>}
+ */
 function determineTargetDir(source, target) {
 	return stat(source).
 		then(results => results.isDirectory() ? target : path.dirname(target));
 }
 
+/**
+ * @param {string[]} fileNames
+ * @param {ProcessFile} config
+ * @returns {Promise<unknown>}
+ */
 function processFiles(fileNames, config) {
 	return Promise.all(fileNames.map(fileName => processFile(fileName, config)));
 }
 
+/**
+ * @param {string} fileName
+ * @param {ProcessFile} opts
+ * @returns {Promise<unknown>}
+ */
 async function processFile(fileName,
-		{ source, target, targetDir, fingerprint, assetManager, plugins }) {
+		{ source, target, targetDir, fingerprint, assetManager, compactors }) {
 	let sourcePath = path.join(source, fileName);
 	let targetPath = path.join(target, fileName);
 
 	try {
 		var content = await readFile(sourcePath); // eslint-disable-line no-var
 	} catch(err) {
+		// @ts-ignore
```

Comment:
You probably want `@ts-expect-error` instead: https://www.stefanjudis.com/today-i-learned/the-difference-ts-ignore-and-ts-expect-error/

In fact, I tend to add the respective error code to provide context for readers: `// @ts-expect-error TS2345` - right now, I can only guess why this was required.

Reply:
Very good point! Fixed in 5240f4accf81b579ff93617e0ffba153542f16a3